src/snowflake/snowpark/modin/plugin/_internal/type_utils.py

```diff
@@ -92,7 +92,6 @@
     (np.half, FloatType()),
     (np.float16, FloatType()),
     (np.float64, DoubleType()),
-    (np.float_, DoubleType()),
```

Comment:
oh, we actually removed np.float_, i thought he message says it is replaced with np.float?

Reply:
`np.float_` is an alias for `np.float64` after numpy 2.x (https://numpy.org/doc/stable/release/2.0.0-notes.html#numpy-2-0-python-api-removals). We specified float64 already in the line right above this.

Older documentation (https://numpy.org/doc/1.26/reference/arrays.scalars.html#numpy.float_) refers to it as an alias of `double`, so I don't think this would cause any behavioral differences.